platform/android/os_android.h

```diff
@@ -187,6 +188,8 @@ class OS_Android : public OS_Unix {
 	void process_gyroscope(const Vector3 &p_gyroscope);
 	void process_touch(int p_what, int p_pointer, const Vector<TouchPos> &p_points);
 	void process_hover(int p_type, Point2 p_pos);
+	void process_doubleTap(Point2 p_pos);
```

Comment:
I think this should be `process_double_tap`, we use snake case for identifiers (unless this is specifically to match an upstream naming).